### ISSUE TITLE
 keras.Sequential model predict get an exception

### ISSUE THREAD

**Issue by shuizhugeju** [SciSharp/TensorFlow.NET#733]
```
            var x = np.array(new float[,] { { 0, 0 }, { 0, 1 }, { 1, 0 }, { 1, 1 } });
            var y = np.array(new float[] { 0, 1, 1, 0 });
            //keras.optimizers.Adam
            var layers = new List<ILayer>() {
                keras.layers.Dense(32,activation:"relu",input_shape:new TensorShape(2)),
                keras.layers.Dense(32,activation:"relu"),

                keras.layers.Dense(64,activation:"relu"),
                keras.layers.Dense(2,activation:"softmax")
            };
            var model = keras.Sequential(layers);
            model.compile(loss: keras.losses.SparseCategoricalCrossentropy(), new Tensorflow.Keras.Optimizers.Adam(), new string[] { "accuracy" });
            model.fit(x, y, batch_size: 2, epochs: 50, verbose: 1);
            

            //var p = model.predict(x);
            var p = model.predict(x, verbose: 1); //get exception
```
   at Tensorflow.TensorDataset.<>c.<.ctor>b__0_0(Tensor x)
   at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
   at Tensorflow.TensorDataset..ctor(Tensor feature, Tensor label)
   at Tensorflow.DatasetManager.from_tensor(Tensor features, Tensor labels)
   at Tensorflow.Keras.Engine.DataAdapters.TensorLikeDataAdapter.slice_inputs(IDatasetV2 indices_dataset, Tensor x, Tensor y)
   at Tensorflow.Keras.Engine.DataAdapters.TensorLikeDataAdapter..ctor(DataAdapterArgs args)
   at Tensorflow.Keras.Engine.DataAdapters.DataHandler..ctor(DataHandlerArgs args)
   at Tensorflow.Keras.Engine.Model.predict(Tensor x, Int32 batch_size, Int32 verbose, Int32 steps, Int32 max_queue_size, Int32 workers, Boolean use_multiprocessing)
   at TFtest.Program.abc() in C:\Users\binbin\source\repos\TFtest\TFtest\Program.cs:line 60

**Comment by Oceania2018** [comment]
Please try latest version [v0.4.0](https://www.nuget.org/packages/TensorFlow.Keras/).
